### PR TITLE
feat: supported Titan Embeddings G1 – Text v1.2

### PR DESCRIPTION
**Pull request by adubovik**

### CODE REVIEW

**Comment by adubovik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-adapter-bedrock-pr-113.nightly-test.deltixhub.io
>E2E tests status: success

File: aidial_adapter_bedrock/deployments.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ def from_deployment_id(
 
 
 class EmbeddingsDeployment(str, Enum):
+    AMAZON_TITAN_EMBED_TEXT_1 = "amazon.titan-embed-text-v1"
```
Comment (roman-romanov-o) [inline aidial_adapter_bedrock/deployments.py]:
```
AMAZON_TITAN_EMBED_TEXT_V1
AMAZON_TITAN_EMBED_TEXT_V2
```

**Comment by roman-romanov-o** [comment]
Minor, but pleasant for usage

Could you pls change 

```
dimensions: Optional[int] = None
```
to

```
dimensions: Optional[Literal[1024, 512, 256]] = None
```

Because if you set `dimensions = 1`, error is vague, and only after visiting a specific document you understand that there are only these options.

**Review by roman-romanov-o — DISMISSED** [review]

File: aidial_adapter_bedrock/deployments.py (hunk 1 above)
Comment (adubovik) [reply aidial_adapter_bedrock/deployments.py]: Indeed, fixed

**Comment by adubovik** [comment]
I have a bad feeling about adding validation for particular values of dimensions like this:

```
dimensions: Optional[Literal[1024, 512, 256]] = None
```

because too many details of a particular model leaks into the adapter. It increases coupling.
 
I think it's better for an adapter to be loose rather than strict whenever it's possible.
 
Suppose the documentation lags behind the real implementation and model is actually supports dimensions=128.
We wouldn't know, but a user could actually know **undocumented feature** , and he could use it via adapter if the types aren't restricted that much.
 
The same goes about extra request parameters - I'm passing the extra fields _as is_ directly to the model, so that the user of the adapter could realise the full potential of the model.

**Review by roman-romanov-o — APPROVED** [review]

**Pull request merged**